scripts/run-dev-dependencies.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -ex
+
+HERE=$(dirname "$0")
+API=cometr
+NETWORK=comet_nw
+"$HERE"/run-dependencies.sh
+
+# From now on, if the user presses Ctrl+C we should teardown gracefully
+trap cleanup INT
+trap cleanup ERR
```

Comment:
This causes `cleanup` to be called twice, which prints spurious errors. Not we actually need `ERR` but we can retain it and avoid errors by replacing both lines with `trap cleanup EXIT` (see build-test-push.sh)